### ISSUE TITLE
Keyboard and mouse not recognized immediately

### ISSUE THREAD

**Issue by marcus-cr** [openrazer/openrazer#1283]
Settings seem to take effect outside of Linux (during POST, BIOS setup, login screens for Linux and Windows). 

During POST it takes 15+ seconds for the keyboard and mouse  to be recognized and allowing input. Only after this time, when both light up to my RGB color settings, am I able to enter BIOS setup or boot any OS. Once an OS does boot it takes an additional 10-15 seconds in order to use either device at login screens. 

Tried unplugging and replugging in devices, resetting the onboard profiles on a Windows machine, loaded default BIOS settings. 

Bit alarmed that this is persistent and affecting boot times, and my ability to use either device at login screens after long period of time. 

Basilisk
- Product number RZ01-02330100
- Model number RZ01-0233

BlackWidow Elite
- Product number RZ03-02621900-R3U1
- Model number RZ03-0262

**Comment by marcus-cr** [comment]
Linux seems to have messed with my powered USB hub. Changed the port and the noted issues are gone, even though none of my Razer devices were plugged into the hub.

Only issue that remains is the Basilisk seems to persist with the color I set.

**Comment by z3ntu** [comment]
> Only issue that remains is the Basilisk seems to persist with the color I set.

Can you please elaborate on that problem?

**Comment by marcus-cr** [comment]
> > Only issue that remains is the Basilisk seems to persist with the color I set.
> 
> 
> 
> Can you please elaborate on that problem? 

I removed OpenRazer but the mouse RGB color persists.

Apologize for the disorganization. The initial issue probably should've been closed, since this is a separate one.

**Comment by z3ntu** [comment]
Please explain the problem, I don't know what you're talking about.

**Comment by marcus-cr** [comment]
This is separate issue. Closing this, opened new ticket (#1290).
